pkgs/by-name/li/libation/package.nix

```diff
@@ -75,10 +79,10 @@ buildDotnetModule rec {
   '';
 
   meta = {
-    changelog = "https://github.com/rmcrackan/Libation/releases/tag/${src.rev}";
+    changelog = "https://github.com/rmcrackan/Libation/releases/tag/v${version}";
```

Comment:
In these cases I'd usually use `src.tag`.

I know that in the mass package migration from rev to tag, `meta.changelog` was not migrated to `src.tag`, but to whatever `src.tag` was set to, in this case `"v${version}"`.

I don't really agree that this is better.

The `releases/tag/` endpoint needs the tag, why not specify the tag directly?

Anyways: if you have strong feelings about this, fine, but otherwise I'd want to have it as `src.tag`